interface/modules/custom_modules/oe-module-weno/templates/facilities.php

```diff
@@ -73,10 +73,13 @@ function activateFacility(){
         }
         
         function downloadPharmacies(){
+            if (!window.confirm(jsText("This download may take several minutes. Do you want to continue?"))) {
```

Comment:
no need to escape literal that goes to dialog in js context. Would translate it via the javascript xl() function though.

Reply:
oops you're right and I've forgotten about the neat translate js function. We could start using it more in twig I'd think!